skbio/diversity/beta/tests/test_unifrac.py

```diff
@@ -8,29 +8,36 @@
 
 from __future__ import absolute_import, division, print_function
 
-from unittest import TestCase, main
 from io import StringIO
-import os
+from unittest import main, TestCase
 
 import numpy as np
-import pandas as pd
+import numpy.testing as npt
 
-from skbio import TreeNode, DistanceMatrix
-from skbio.util import get_data_path
+from skbio import TreeNode
 from skbio.tree import DuplicateNodeError, MissingNodeError
 from skbio.diversity.beta import unweighted_unifrac, weighted_unifrac
+from skbio.diversity.beta._unifrac import (_boundary_case,
+                                           _unweighted_unifrac_pdist_f,
+                                           _weighted_unifrac_pdist_f,
+                                           _unweighted_unifrac,
+                                           _weighted_unifrac,
+                                           _weighted_unifrac_branch_correction)
 
 
-class TestUniFrac(TestCase):
+class UnifracTests(TestCase):
```

Comment:
Confirm that the ``beta_diversity`` driver is hitting the same kinds of tests as the ``alpha_diversity`` driver.